apps/verticalhome/test/marionette/grid_layout_test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var assert = require('assert');
+
+marionette('Vertical - grid', function() {
+
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var actions, home, system;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    home = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+    home.waitForLaunch();
+  });
+
+  test('apps grid is LTR in LTR language', function() {
+    var communicationManifestUrl =
+        'app://communications.gaiamobile.org/manifest.webapp';
+    var phoneIcon = home.getIcon(communicationManifestUrl, 'dialer').location();
+    var contactsIcon = home.getIcon(communicationManifestUrl, 'contacts')
+      .location();
+    var messageIcon = home.getIcon('app://sms.gaiamobile.org/manifest.webapp')
+      .location();
```

Comment:
> Side-note, 'aligned to the left' is probably best tested as x < (gridWidth/columns), to account for padding (and where I assume padding will never be so huge as to be greater than the grid item width).

Actually here I test the order of the icons (respective to x coordinates) and not the fact that they are left-aligned. It seems sensible to me because that what matters to users anyway, and also because we don't care about padding this way. What do you think?